perun-base/pom.xml

```diff
@@ -136,6 +136,13 @@
 			<artifactId>javax.mail</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20140107</version>
```

Comment:
Can you please use the current version ? I found latest to be ``20190722``. Maybe check, if our parent pom ``spring-boot-starter-parent`` doesn't specify it already.

Reply:
Changed to 20190722.